packages/strapi-plugin-content-manager/admin/src/containers/EditPage/saga.js

```diff
@@ -88,8 +89,18 @@ export function* submit() {
     yield put(setLoader());
     const recordCleaned = Object.keys(record).reduce((acc, current) => {
       const attrType = source !== 'content-manager' ? get(schema, ['models', 'plugins', source, currentModelName, 'fields', current, 'type'], null) : get(schema, ['models', currentModelName, 'fields', current, 'type'], null);
-      const cleanedData = attrType === 'json' ? record[current] : cleanData(record[current], 'value', 'id');
-
+      let cleanedData;
+
+      switch (attrType) {
+        case 'json':
+          cleanedData = record[current];
+          break;
+        case 'date':
+          cleanedData = moment(record[current]).format();
```

Comment:
I haven't specified any format here and I don't know if I should specify one...
@lauriejim @Aurelsicoko do you confirm that it matches what the back-end is expecting?

Reply:
In terms of bookshelf at least with MySQL, the format *has* to be in `YYYY-MM-DD HH:MM:SS` or the knex create/update query will fail.

Reply:
@derrickmehaffy This PR fixes the time and date picker that were unusable in the front-end.
I have tested it with mongo and I know that it works.

Regarding bookshelf I think that it should be the back-end that should take care of formatting the received date instead of the front-end that way users won't have to duplicate the admin logic in their client app.

**cc/** @lauriejim